src/utils/workflows/getWorkflowSteps.ts

```diff
@@ -1,9 +1,19 @@
 import { ZObject } from 'zapier-platform-core';
 import { KontentBundle } from '../../types/kontentBundle';
 import { createManagementClient } from '../kontentServices/managementClient';
+import { WorkflowContracts, WorkflowModels } from '@kontent-ai/management-sdk';
+import { nullMap } from '../functional';
 
-export const getWorkflowSteps = async (z: ZObject, bundle: KontentBundle<{}>) =>
-  createManagementClient(z, bundle)
-    .listWorkflowSteps()
-    .toPromise()
-    .then(res => res.data);
+export const getWorkflow = async (z: ZObject, bundle: KontentBundle<{}>): Promise<WorkflowModels.Workflow> =>
```

Comment:
cant his be just promise? and using .catch?

Reply:
Since I search for the default WF using `.find`, I need to check the result is not `undefined` to narrow TS types. I don't think it is necessary to change it.